TechTalk.JiraRestClient/Compatibility.cs

```diff
@@ -12,7 +12,9 @@ public interface IJiraClient
         /// <summary>Returns all issues of the specified type for the given project</summary>
         IEnumerable<Issue> GetIssues(String projectKey, String issueType);
         /// <summary>Returns all issues of the given type and the given project filtered by the given JQL query</summary>
-        IEnumerable<Issue> GetIssuesByQuery(String projectKey, String issueType, String jqlQuery);
+        IEnumerable<Issue> GetIssuesByQuery(String projectKey, String issueType, int startAt = 0);
```

Comment:
Breaking change: don't remove interface methods
I wouldn't add 'startAt' to this overload; it is a rather low level feature